budgeting_tool/main.js

```diff
@@ -0,0 +1,114 @@
+/* Instructions:
+
+1. Wire up the budgeting tool so it tells the user whether or 
+not they can afford an item based on their available funds. 💸
+2. Style it as you wish 💅 
+
+*/
+
+const fundTextElemnt = document.getElementById("fund-text")
+const fundElement = document.getElementById("fund");
+const itemCostElement = document.getElementById("item-cost");
+const itemElement = document.getElementById("item");
+const calculationBtn = document.getElementById("btn-calculation");
+const extraFundAmountElement = document.getElementById("extra-fund-amount");
+const extraFundTextElement = document.getElementById("extra-fund-text");
+const extraFundBtn = document.getElementById("btn-extra-fund");
+const extraFundElements = document.getElementById("extra-fund");
+const resultElement = document.getElementById("result");
+const currentdataElement = document.getElementById("current-data");
+
+const totalCostElement = document.getElementById("total-cost");
+const remainingMoneyElement = document.getElementById("remaining-money")
+
+const DEFAULT_VALUE = 0;
+let fund = DEFAULT_VALUE;
+let remainingMoney = DEFAULT_VALUE;
+let itemCost = DEFAULT_VALUE;
+let TotalCost = DEFAULT_VALUE;
+
+calculationBtn.addEventListener("click", checkFundInput);
+extraFundBtn.addEventListener("click", checkExtraFund);
+
+
+
+function addList(itemElementValue, itemCost){
+    const p = document.createElement("p");
+    p.innerHTML = `${itemElementValue}: ${itemCost}`;
+    p.classList.add("item-list");
+    extraFundElements.appendChild(p);
+}
+
+function valueFormatted(value){
+    return parseFloat(value).toLocaleString('en-GB', {minimumFractionDigits: 2, maximumFractionDigits: 2});
+}
+
+function ExtraFundShow(fund, TotalCost, remainingMoney){  
```

Comment:
Although there are no naming convention rules for JS, for variables and functions, camelCase is widely accepted as a standard, e.g. extraFunds()